From 72bb1a69cf6c0f58d3c1a8f6ba98334640818566 Mon Sep 17 00:00:00 2001
From: Paul Kocialkowski <contact@paulk.fr>
Date: Sun, 9 Aug 2015 12:06:28 +0200
Subject: [PATCH 3/7] DOTCONFIG location correction

The configuration file doesn't have to be in src and HAVE_DOTCONFIG holds its
current location.

Signed-off-by: Paul Kocialkowski <contact@paulk.fr>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index a1a9d33..13305cd 100644
--- a/Makefile
+++ b/Makefile
@@ -73,7 +73,7 @@ all: help
 
 else
 
-include $(src)/.config
+include $(HAVE_DOTCONFIG)
 
 ifeq ($(CONFIG_ARCH_X86),y)
 ARCH = x86
-- 
1.9.1

